packages/apps-config/src/ui/logos/index.ts

```diff
@@ -78,6 +79,8 @@ const chainLogos: Record<string, any> = [
 // alphabetical
 const nodeLogos: Record<string, any> = [
   ['airalab-robonomics', nodeRobonomics],
+  ['Bifrost Node', nodeBifrost],
+  ['Bifrost', nodeStafi],
```

Comment:
```suggestion
  ['Bifrost', nodeStafi],
```

I believe this is incorrect?

Reply:
@jacogr  Thank you very much.